### PR TITLE
Separated the 'set' help commands. 'set' now returns all commands ava…

### PR DESCRIPTION
**Pull request by Elyriand21**
- Moved all of the available 'set' commands to be displayed when 'set' is typed.
- Cleaned up the 'help' command

### CODE REVIEW

File: DeveloperConsole.cs [hunk 1]
```diff
@@ -40,13 +40,23 @@ private void CommandRunner (string command)
                     result = "Commands:\n" +
                         "help - Displays this message\n" +
                         "clear - Clears the command history\n" +
-                        "set - Sets a value\n" +
-                        "set health <value> - Sets the player's health\n";
+                        "set - Shows the commands available for set\n";
```
Comment (Lillious) [inline DeveloperConsole.cs]: change to "help set - Shows the commands available for set\n";

File: DeveloperConsole.cs [hunk 2]
```diff
@@ -40,13 +40,23 @@ private void CommandRunner (string command)
...
                     break;
+
+                // Set command
                 case "set":
```
Comment (Lillious) [inline DeveloperConsole.cs]: This section is for the command control, not the help section. If the intention is to add a help section for set commands, please create a secondary switch case inside help

**Review by Lillious — CHANGES_REQUESTED** [review]
result already has \n appended to it. Please revert the \n from all result lines.
Help commands will want to be structured as help set, not just running the set command as that is reserved for an invalid arguments condition.

**Comment by Elyriand21** [comment]
This should have fixed the issues.

The // Set Command on LINE 59 is the same concept as the // Help Command on LINE 39. It only indicates the section that is being worked. LINE 49 is the "help set" command

File: DeveloperConsole.cs [hunk 3]
```diff
@@ -100,6 +116,7 @@ private void Initialize()
             if (!isInGame) return;
             initialized = true;
             Player = LC_Lib.GetPlayer("Player");
+            Terminal = LC_Lib.GetTerminal();
```
Comment (Lillious) [inline DeveloperConsole.cs]: GetTerminal method does not exist on LC_Lib

File: DeveloperConsole.cs [hunk 4]
```diff
@@ -21,12 +21,13 @@ public class DeveloperConsole : MelonMod
         private Canvas DevConsole;
         static Library LC_Lib = new Library();
         PlayerControllerB Player;
+        Terminal Terminal;
```
Comment (Lillious) [inline DeveloperConsole.cs]: Terminal does not exist in context

**Review by Lillious — CHANGES_REQUESTED** [review]
Review individual comments

File: DeveloperConsole.cs (hunk 3 above)
Comment (Elyriand21) [reply DeveloperConsole.cs]: This should be fixed due to the push to the Library. Mod-Library now includes a "GetTerminal" method

File: DeveloperConsole.cs (hunk 4 above)
Comment (Elyriand21) [reply DeveloperConsole.cs]: I'm not exactly following. Upon building the code, it finds the Terminal GameObject and compiles it.

**Review by Elyriand21 — COMMENTED** [review]
I believe that the "GetTerminal" method has been resolved due to the push to the Library

**Pull request closed without merging**